### PR TITLE
Allow updating `quoteType` and `isValueless` of AttrNodes, and `quoteType` of StringLiterals

### PR DESCRIPTION
**Pull request by courajs**
Enhancement: allow setting `quoteType` of AttrNodes during transform.

This enables adding a fixer for the `quotes` rule to `ember-template-lint`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -125,7 +125,9 @@ export interface TransformPluginBuilder {
 ```
 
 The list of known builders on the `env.syntax.builders` are [found
-here](https://github.com/glimmerjs/glimmer-vm/blob/v0.62.4/packages/%40glimmer/syntax/lib/builders.ts#L547-L578).
+here](https://github.com/glimmerjs/glimmer-vm/blob/v0.62.4/packages/%40glimmer/syntax/lib/builders.ts#L547-L578),
```
Comment (rwjblue) [inline README.md:128]: We are now using 0.82, mind updating the link?

**Review by rwjblue — APPROVED** [review]
This is great work @courajs! Thank you for working on it!!!

**Comment by courajs** [comment]
Changed link, and fixed merge conflicts!

**Comment by courajs** [comment]
Also fixes #587 and #588

**Review by dcyriller — APPROVED** [review]

**Comment by dcyriller** [comment]
Released in v6.1.0 🎉

**Pull request merged**